apps/spruce/src/pages/distroSettings/tabs/GeneralTab/getFormSchema.ts

```diff
@@ -84,6 +95,12 @@ export const getFormSchema = (
         }),
       },
     },
+    distroImage: {
+      "ui:ObjectFieldTemplate": CardFieldTemplate,
+      identifier: {
```

Comment:
Should this be id for consistency? Since we call the field `imageId`

Reply:
ID is short for identity/identifier so I don't think it matters much. The distro ID itself is called identifier right above this line